packages/create-plugin/fixtures/test-template/src/plugin.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
this fixture is to replace an existing failing test due to changes in the plugin.json